CMakeLists.txt

```diff
@@ -90,6 +90,13 @@ SET_SOURCE_FILES_PROPERTIES(src/fish_version.cpp
                             PROPERTIES OBJECT_DEPENDS
                             ${CMAKE_CURRENT_BINARY_DIR}/${FBVF})
 
+OPTION(ENABLE_INTERNAL_WCWIDTH "use fallback wcwidth" ON)
```

Comment:
Can I suggest this is just "INTERNAL_WCWIDTH" in line with other options?

Reply:
Done.